rst2pdf/styles.py

```diff
@@ -610,7 +611,11 @@ def readStyle(self, ssname):
         fname = self.findStyle(ssname)
         if fname:
             try:
-                return rson_loads(open(fname).read())
+                # Is it an older rson/json stylesheet with .style extension?
+                if fname[-5:] == "style":
+                    return rson_loads(open(fname).read())
```

Comment:
Could we log a warning here, indicating that users should convert their style files to YAML file?

Reply:
Great idea, added!